README.md

```diff
@@ -348,10 +331,7 @@ be changed to latest.
 
 ####`package_gpg_key`
 
-RPM package GPG key to import. Uses source method. Should be a URL for Debian/RedHat
-OS family, or a file name for RedHat OS family.
-Set to http://www.rabbitmq.com/rabbitmq-signing-key-public.asc by default.
-Note, that `key_content`, if specified, would override this parameter for Debian OS family.
+This should generally be left as default, unless you know what you're doing.
```

Comment:
This should contain documentation at least describing one use case in which it could be used.